plugins/woocommerce/includes/data-stores/class-wc-order-data-store-cpt.php

```diff
@@ -104,7 +105,7 @@ public function create( &$order ) {
 		parent::create( $order );
 
 		// Do not fire 'woocommerce_new_order' for draft statuses.
-		if ( in_array( $order->get_status( 'edit' ), array( 'auto-draft', 'draft', 'checkout-draft' ), true ) ) {
+		if ( in_array( $order->get_status( 'edit' ), array( OrderStatus::AUTO_DRAFT, OrderStatus::DRAFT, 'checkout-draft' ), true ) ) {
```

Comment:
If `checkout-draft` is a possibility, should it be added to the enumeration class? Not immediately sure about this, but wanted to pause and ask.